Could not find artifact net.minidev:json-smart:jar:2.3

I believe we're suffering from this issue:

- https://github.com/netplex/json-smart-v2/issues/240

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.414 s
[INFO] Finished at: 2025-02-12T09:33:04-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project dataverse: Could not resolve dependencies for project edu.harvard.iq:dataverse:war:6.5
[ERROR] dependency: net.minidev:json-smart:jar:2.3 (compile)
[ERROR] 	Could not find artifact net.minidev:json-smart:jar:2.3 in unidata-all (https://artifacts.unidata.ucar.edu/repository/unidata-all/)
```

The workaround at https://github.com/apache/camel/commit/f12f6269d4ca12ecec6aa705e4c05b9812543f80 seems decent.

--- a/pom.xml
+++ b/pom.xml
@@ -494,7 +494,7 @@
         <dependency>
             <groupId>com.nimbusds</groupId>
             <artifactId>oauth2-oidc-sdk</artifactId>
-            <version>10.13.2</version>
+            <version>11.22.1</version>
         </dependency>
         <!-- Caching library, current main use case is for OIDC authentication -->
         <dependency>

